css/properties/flex.json

```diff
@@ -0,0 +1,109 @@
+{
+  "css": {
+    "properties": {
+      "flex": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/flex",
+          "support": {
+            "webview_android": {
+              "version_added": "4.4"
+            },
+            "chrome": [
+              {
+                "version_added": "29"
+              },
+              {
+                "prefix": "-webkit-",
+                "version_added": "21"
+              }
+            ],
+            "chrome_android": {
+              "version_added": null
+            },
+            "edge": [
+              {
+                "version_added": true
+              },
+              {
+                "prefix": "-webkit-",
+                "version_added": true
+              }
+            ],
+            "edge_mobile": [
+              {
+                "version_added": true
+              },
+              {
+                "prefix": "-webkit-",
+                "version_added": true
+              }
+            ],
+            "firefox": [
+              {
+                "version_added": "20",
+                "notes": [
+                  "Since Firefox 28, multi-line flexbox is supported.",
+                  "Before Firefox 32, Firefox wasn't able to animate values starting or stopping at <code>0</code>."
+                ]
+              },
+              {
+                "prefix": "-webkit-",
+                "version_added": "49",
+                "notes": "For Firefox 48, the <code>-webkit-</code> prefix was available with the <code>layout.css.prefixes.webkit</code> preference. Starting with Firefox 49, the preference defaults to <code>true</code>."
+              },
+              {
+                "version_added": "18",
+                "version_removed": "20",
+                "flag": {
+                  "type": "preference",
+                  "name": "layout.css.flexbox.enabled",
+                  "value_to_set": "true"
+                }
+              }
+            ],
+            "ie": [
+              {
+                "version_added": "11",
+                "notes": [
+                  "Internet Explorer 10 and 11 ignore uses of <a href='https://developer.mozilla.org/docs/Web/CSS/calc'><code>calc()</code></a> in the <code>flex-basis</code> part of the <code>flex</code> syntax. This can be worked around by using the longhand properties instead of the shorthand. See <a href='https://github.com/philipwalton/flexbugs#8-flex-basis-doesnt-support-calc'>Flexbug #8</a> for more info.",
```

Comment:
Support is in "11" but notes apply to "10" too. Which one is incorrect?

Reply:
I think this note refers to the behavior of the `-ms-` prefixed and the unprefixed properties. I've broken them up into separate (though very similar) notes.